File: 4354-unique-middle-element/unique-middle-element.py
class Solution:
    def isMiddleElementUnique(self, nums):
        mid = len(nums) // 2
        target = nums[mid]

        count = 0

        for num in nums:
            if num == target:
                count += 1

                if count > 1:
                    return False

        return True

File: 4354-unique-middle-element/unique-middle-element.java
class Solution {
    public boolean isMiddleElementUnique(int[] nums) {

        int mid = nums.length / 2;
        int target = nums[mid];

        int count = 0;

        for (int num : nums) {

            if (num == target) {
                count++;

                if (count > 1) {
                    return false;
                }
            }
        }

        return true;
    }
}